tests/FixturesTraitTest.php

```diff
@@ -167,4 +167,13 @@ public function testPrepareSequences()
 
         $this->prepareSequences($this->getTables());
     }
+
+    public function testGetFixtureWithoutGlobalExportMode()
+    {
+        $response = $this->getJsonFixture('get_fixture/export_fixture.json');
```

Comment:
let's also use any existed fixture without creating new one

```suggestion
        $content = $this->getJsonFixture('get_fixture/export_fixture.json');
```